cfme/middleware/__init__.py

```diff
@@ -27,6 +31,17 @@ def _on_detail_page(self):
         return sel.is_displayed('//h1[contains(., "{} (Summary)")]'.format(self.name))
 
 
+import_form = Form(
+    fields=[
+        ("file_select", "#upload_file"),
```

Comment:
FYI, FileInput nad Input classes take only the "name" of the field, so in this case, `FileInput('upload_file')` ...